docs/api/migration-guides/qiskit-1.0-features.mdx

```diff
@@ -0,0 +1,600 @@
+---
+title: Qiskit 1.0 feature migration guide
+description: Adapt to feature changes with Qiskit 1.0
+---
+
+# Qiskit 1.0 feature changes
+
+Qiskit 1.0 ...
+
+This guide is divided ...
+
+* Changes to [global instances and functions](#qiskit.global) in qiskit
+* Changes to [`qiskit.circuit`](#qiskit.circuit) module
+* Changes to [`qiskit.converters`](#qiskit.converters) module
+* Changes to [`qiskit.extensions`](#qiskit.extensions) module
+* Changes to [`qiskit.primitives`](#qiskit.primitives) module
+* Changes to [`qiskit.providers`](#qiskit.providers) module
+* Changes to [`qiskit.pulse`](#qiskit.pulse) module
+* Changes to [`qiskit.quantum_info`](#qiskit.quantum_info) module
+* Changes to [`qiskit.synthesis`](#qiskit.synthesis) module
+* Changes to [`qiskit.test`](#qiskit.test) module
+* Changes to [`qiskit.tools`](#qiskit.tools) module
+* Changes to [`qiskit.transpiler`](#qiskit.transpiler) module
+* Changes to [`qiskit.utils`](#qiskit.utils) module
+* Changes to [`qiskit.visualization`](#qiskit.visualization) module
+
+
+<span id="qiskit.global"></span>
+## Changes to global instances and functions
+
+* The `qiskit.Aer` object is no longer available in Qiskit 1.0.  You should instead
+  use the same object from the `qiskit_aer` namespace, which is a drop-in replacement.
+  To install `qiskit_aer`, run:
+
+  ```bash
+  pip install qiskit-aer
+  ```
+* The `qiskit.BasicAer` object is no longer available in Qiskit 1.0. See the
+  [`basicaer migration section`](#migrate-from-basicaer-to-basic_provider) for more
+  information on migration options.
+
+* The `qiskit.execute` function is no longer available in Qiskit 1.0.
+  This function served as a high-level wrapper around the transpile and run functionalities
+  in Qiskit. Instead of `qiskit.execute`, you can explicitly use the `transpile` function
+  (with appropriate transpile options) followed by `backend.run()`
+  (with appropriate run options).
+
+  Migration example:
+
+  ```python
+  # Legacy path
+  from qiskit import execute
+  job = execute(circuit, backend)
+
+  # New path
+  from qiskit import transpile
+  new_circuit = transpile(circuit, backend)
+  job = backend.run(new_circuit)
+  ```
+
+  Alternatively, the `Sampler` primitive is semantically equivalent to the
+  deprecated `qiskit.execute` function. The class `BackendSampler`
+  is a generic wrapper for backends that do not support primitives:
+
+  ```python
+  from qiskit.primitives import BackendSampler
+  sampler = BackendSampler(backend)
+  job = sampler.run(circuit)
+  ```
+
+<span id="qiskit.circuit"></span>
+## Changes to `qiskit.circuit`
```

Comment:
Would you also like to mention the removal of the duplicated gate methods?

    * ``QuantumCircuit.cnot``, instead use :meth:`.QuantumCircuit.cx`,
    * ``QuantumCircuit.toffoli``, instead use :meth:`.QuantumCircuit.ccx`,
    * ``QuantumCircuit.fredkin``, instead use :meth:`.QuantumCircuit.cswap`,
    * ``QuantumCircuit.mct``, instead use :meth:`.QuantumCircuit.mcx`,
    * ``QuantumCircuit.i``, instead use :meth:`.QuantumCircuit.id`.